src/BooleanSlider/widget/BooleanSlider.js

```diff
@@ -117,8 +117,8 @@ define([
             if (this._contextObj !== null) {
                 dojoStyle.set(this.domNode, "display", "block");
 
-                dojoProp.set(this.trueNode, "textContent", this.trueValue);
-                dojoProp.set(this.falseNode, "textContent", this.falseValue);
```

Comment:
The next one I am trying to go to tackle is the Share buttons. That one is for Mx3 and has not been updated since 2013. I have 12 lines in my deprecation xls so it needs some rework :)